index.html

```diff
@@ -203,6 +203,75 @@ <h4 class="skillCardHeader">Skills</h4>
         </div>
       </div>
     </section>
-    <section class="contactMe"></section>
+    <footer class="contactMe">
+      <header><div class="headerv"></div></header>
+      <section id="formSection">
+        <h2>Get started</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/xbjbqokr" method="POST">
+          <ul>
+            <li>
+              <div class="inputDiv">
+                <label for="name">Your name</label>
+                <input
+                  type="text"
+                  name="name"
+                  maxlength="30"
+                  required="true"
+                  placeholder="Type your name here..."
+                />
+              </div>
+            </li>
+            <li>
+              <div class="inputDiv">
+                <label for="name">Your email</label>
+                <input
+                  type="email"
+                  name="email"
+                  required="true"
+                  placeholder="Type your email here..."
+                />
+              </div>
+            </li>
+            <li>
+              <textarea
+                name="message"
+                required="true"
+                id="message"
+                maxlength="500"
+                placeholder="Enter your message here..."
+                rows="5"
+              ></textarea>
+            </li>
+            <li>
+              <button type="submit" class="btn startCoText">
+                Start Collaboration
+              </button>
+            </li>
+          </ul>
+        </form>
+      </section>
+      <footer id="contactFooter">
+        <div id="footerContainer">
+          <button class="myResume">
+            <div class="btnTextResume">Get my Resume</div>
+            <div class="btnDownload">
+              <img src=".png/ic_link_teal.png" alt="download-icon" />
+            </div>
+          </button>
+          <div id="links">
+            <a href=""><img src=".png/ic_stackoverflow.png" alt=""></a>
+            <a href=""><img src=".png/ic_github.png" alt=""></a>
+            <a href=""><img src=".png/ic_twitter.png" alt=""></a>
+            <a href=""><img src=".png/ic_linkedin.png" alt=""></a>
+            <a href=""><img src=".png/ic_medium.png" alt=""></a>
+            <a href=""><img src=".png/ic_angellist.png" alt=""></a>
+          </div>
```

Comment:
- [ ] Kindly use `ul` and `li` tags instead of using div tag due to SEO purpose :blush: as shown below ⬇️
```

          <ul id="links">
            <li><a href=""><img src=".png/ic_stackoverflow.png" alt=""></a></li>
            <li><a href=""><img src=".png/ic_github.png" alt=""></a></li>
            <li><a href=""><img src=".png/ic_twitter.png" alt=""></a></li>
            <li><a href=""><img src=".png/ic_linkedin.png" alt=""></a></li>
            <li><a href=""><img src=".png/ic_medium.png" alt=""></a></li>
            <li><a href=""><img src=".png/ic_angellist.png" alt=""></a></li>
          </ul>
```